src/main/java/code401challenges/linkedlist/LinkedList.java

```diff
@@ -128,17 +124,44 @@ public int size(){
     }
 
 
+
     public int kthFromTheLast(int k){
         int size = this.size();
         if(k > size){
             throw new NullPointerException("Not a valid node index, list size is " + size);
         }
         Node currNode = this.head;
-       int stepper = 1;
+       int stepper = 0;
        while(stepper < k){
            stepper++;
            currNode = currNode.next;
        }
         return currNode.value;
     }
+
+
+
+    public static LinkedList mergeLists(LinkedList one, LinkedList two){
+        //create variables to hold each nodes. next
+        Node cnl1 = one.head;
+        Node cnl2 = two.head;
+        Node cnl1Next = cnl1.next;
+        Node cnl2Next = cnl2.next;
+
+//        check if any of the current next values are null
+        if(cnl1Next != null) {cnl1 = one.head.next;}
+        if(cnl2Next != null) {cnl2 = two.head.next;}

```

Comment:
Aren't you doing this already on the lines above?  .... where you write: Node cnl1Next = cnl1.next;